core/templates/dev/head/components/top_navigation_bar/TopNavigationBarDirective.js

```diff
@@ -67,24 +82,61 @@ oppia.directive('topNavigationBar', [
             $window.localStorage.removeItem('last_uploaded_audio_lang');
           };
 
-          $scope.profileDropdownIsActive = false;
-          $scope.onMouseoverProfilePictureOrDropdown = function(evt) {
-            angular.element(evt.currentTarget).parent().addClass('open');
-            $scope.profileDropdownIsActive = true;
-          };
-          $scope.onMouseoutProfilePictureOrDropdown = function(evt) {
```

Comment:
Great to see this removed, but make sure you also grep the code for anything that calls it and deal with that accordingly as well.

Reply:
Done.
There is another issue regarding navigation in admin page, the tabbing order is reverse in admin navigation bar. Basically, when we press tab, we expect the focus to be shifted to next element, but in admin navigation bar, the focus is shifting to the previous element.Shall I work on that too?